example/linux/systemd/Talawa-api.sh

```
@@ -0,0 +1,157 @@
+  echo "Error: Production path '$PROD_PATH' not found. Exiting." | tee -a "$LOG_FILE"
+  exit 1
+fi
+
+echo "Development and production paths are valid. Proceeding..." | tee -a "$LOG_FILE"
+
+# Check if .env file is present
+if [ ! -f ".env" ]; then
+  echo "Error: '.env' file not found. Exiting." | tee -a "$LOG_FILE"
+  exit 1
+fi
+echo ".env file found in "$pwd" directory. Proceeding..." | tee -a "$LOG_FILE"
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)